### PR TITLE
fix(frontend): shifted actual data 5 seconds forward

### PR DESCRIPTION
**Pull request by shaleynikov**
…fixing almost all issues with selection

Related to:
- https://github.com/pyroscope-io/pyroscope/issues/621
- https://github.com/pyroscope-io/pyroscope/issues/713

### CODE REVIEW

File: webapp/javascript/components/TimelineChartWrapper.jsx [hunk 1]
```diff
@@ -157,10 +157,28 @@ class TimelineChartWrapper extends React.Component {
...
+          this.props.timeline.map((x) => [
+            x[0] + 5000,
+            x[1] === 0 ? null : x[1] - 1,
+          ]),
+        ]
       : [];
 
+    // In case there are few chunks left, then we'd like to add some margins to
+    // both sides making it look more centers
+    const flotOptions = {
```
Comment (petethepig) [inline webapp/javascript/components/TimelineChartWrapper.jsx:174]: You're not actually using this one later on line :187


Other than that it looks and works great!

**Review by petethepig — COMMENTED** [review]
see comments above

**Review by petethepig — CHANGES_REQUESTED** [review]
see comments above

**Comment by shaleynikov** [comment]
Now it should be fine :-)

**Review by shaleynikov — COMMENTED** [review]
Now it uses options provided

**Review by petethepig — APPROVED** [review]
Looks great, thank you!

**Pull request merged**